### ISSUE TITLE
Fix dependabot errors

### ISSUE THREAD

**Issue by MrDeshaies** [canada-ca/aia-eia-js#843]
GitHub's dependabot has identified a number of libraries with security vulnerabilities.

Upgrade of dependent libraries which pose at “moderate severity” or above to a version that addresses the vulnerability, as identified by dependabot (https://github.com/canada-ca/aia-eia-js/security/dependabot). As of 2021-09-27, there were 15 alerts.

**Comment by MrDeshaies** [comment]
As of now, there are 5 alerts left, but I note that 1 was created after @Ancentury  started with us (technically out of scope). N.B. may be blocked by issue #841 ; not sure if dependabot refreshes when the build fails.
